frontend/lib/requests/galleries.ts

```
@@ -0,0 +1,47 @@
+"use server";
+
+import { Enrollment, Gallery } from "@/types";
```

ESLint

[error] 'Enrollment' is defined but never used.

(@typescript-eslint/no-unused-vars)